rbc/tests/heavydb/test_timestamp.py

```diff
@@ -0,0 +1,48 @@
+from packaging.version import Version, InvalidVersion
+
+from rbc.tests import heavydb_fixture
+import pytest
+
+rbc_heavydb = pytest.importorskip('rbc.heavydb')
+available_version, reason = rbc_heavydb.is_available()
+pytestmark = pytest.mark.skipif(not available_version, reason=reason)
+
+try:
+    curr_version = Version('.'.join(map(str, available_version[:2])))
+except InvalidVersion:
+    pytest.skip("HeavyDB unavailable", allow_module_level=True)
+else:
+    if curr_version < Version("6"):
+        pytest.skip(
+            f"Requires HeavyDB >= 6, got {available_version}",
+            allow_module_level=True
+        )
+
+
+@pytest.fixture(scope='module')
+def heavydb():
+    for o in heavydb_fixture(globals(), debug=not True, load_columnar=True):
```

Comment:
`heavydb_fixture` accepts an argument `minimal_version` which you can use to skip the whole file if the server version is not what you expect.

```suggestion
@pytest.fixture(scope='module')
def heavydb():
    for o in heavydb_fixture(globals(), debug=not True, load_columnar=True, minimal_version=(6, 0)):
```

And inside `rbc/tests/heavydb/__init__.py::heavydb_fixture`, there's a function called `require_version` which checks if the given version is older than the available one. Move what you wrote here to there.

Reply:
Aoutch seeing this... I will move this in for now, but this whole thing deserves a rewrite at some point.

Reply:
You mean the fixture? I've changed it a bit on #496